src/core/render-biblio.js

```diff
@@ -158,7 +158,7 @@ function toRefContent(ref) {
 export function renderInlineCitation(ref) {
   const key = ref.replace(/^(!|\?)/, "");
   const href = `#bib-${key.toLowerCase()}`;
-  return hyperHTML`[<cite><a class="bibref" href="${href}">${key}</a></cite>]`;
+  return hyperHTML`[<cite><a class="biblio" href="${href}">${key}</a></cite>]`;
```

Comment:
The task is to add a `data-link-type="biblio"`, not to change the class name.

Reply:
> The task is to add a `data-link-type="biblio"`, not to change the class name.

okay! i'll try to fix that